docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '3.8'
+
+services:
+  db:
+    image: postgres:12
+    ports:
+    - "5432:5432"
+    restart: always
+    environment:
+      POSTGRES_DB: ${DB_NAME}
+      POSTGRES_USER: ${DB_USER}
+      POSTGRES_PASSWORD: ${DB_PASS}
+    volumes:
+    - "db-data:/var/lib/postgres"
+
+  app:
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    ports:
+    - "8081:8081"
+    restart: always
+    depends_on:
+    - db
+    environment:
+      JDBC_URL: jdbc:postgresql://db:5432/${DB_NAME}
+      JDBC_USERNAME: ${DB_USER}
+      JDBC_PASSWORD: ${DB_PASS}
+      SPRING_PROFILES_ACTIVE: prdtest
```

Comment:
Noticed this is defaulted to prdtest rather than prd. If this is what you wanted, feel free to ignore this comment!